wallet-demo-app/kustomize/wallet-demo-app/overlays/local/wallet-demo-app/ingress.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright Gen Digital Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: wallet-demo-app
+  labels:
+    app: wallet-demo-app
+spec:
+  tls:
+  - hosts:
+      - wallet-demo-app.||DOMAIN||
+    secretName: INGRESS_TLS_SECRET
+  rules:
+  - host: wallet-demo-app.||DOMAIN||
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/"
+        backend:
+          service:
+            name: wallet-demo-app-service-name
```

Comment:
this should match the service name from base: [wallet-demo-app](https://github.com/trustbloc/k8s/pull/671/files#diff-fae09a39dac92209fd2abfcd2cc461123a4dd995eca6c64e4e1b5a452752af99)